pkg/cmd/logs.go

```diff
@@ -0,0 +1,254 @@
+		Use:   "labels",
+		Short: "Get labels of a tenant.",
+		Long:  "Get labels of a tenant.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			f, currentTenant, err := fetcher.NewCustomFetcher(ctx, logger)
+			if err != nil {
+				return fmt.Errorf("custom fetcher: %w", err)
+			}
+
+			params := &client.GetLogLabelsParams{}
+			if labelStart != "" {
+				params.Start = (*parameters.StartTS)(&labelStart)
+			}
+			if labelEnd != "" {
+				params.End = (*parameters.EndTS)(&labelEnd)
+			}
+
+			resp, err := f.GetLogLabelsWithResponse(ctx, currentTenant, params)
+			if err != nil {
+				return fmt.Errorf("getting response: %w", err)
+			}
+
+			return handleResponse(resp.Body, resp.HTTPResponse.Header.Get("content-type"), resp.StatusCode(), cmd)
+		},
+	}
+
+	labelsCmd.Flags().StringVarP(&labelStart, "start", "s", "", "Start timestamp.")
+	labelsCmd.Flags().StringVarP(&labelEnd, "end", "e", "", "End timestamp.")
+
+	// Labelvalues command.
+	var (
+		labelName, labelValuesStart, labelValuesEnd string
+	)
+	labelValuesCmd := &cobra.Command{
+		Use:          "labelvalues",
+		Short:        "Get label values of a tenant.",
+		Long:         "Get label values of a tenant.",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			f, currentTenant, err := fetcher.NewCustomFetcher(ctx, logger)
+			if err != nil {
+				return fmt.Errorf("custom fetcher: %w", err)
+			}
+
+			params := &client.GetLogLabelValuesParams{}
+
+			if labelValuesStart != "" {
+				params.Start = (*parameters.StartTS)(&labelValuesStart)
+			}
+			if labelValuesEnd != "" {
+				params.End = (*parameters.EndTS)(&labelValuesEnd)
+			}
+
+			resp, err := f.GetLogLabelValuesWithResponse(ctx, currentTenant, labelName, params)
+			if err != nil {
+				return fmt.Errorf("getting response: %w", err)
+			}
+
+			return handleResponse(resp.Body, resp.HTTPResponse.Header.Get("content-type"), resp.StatusCode(), cmd)
+		},
+	}
+	labelValuesCmd.Flags().StringVar(&labelName, "name", "", "Name of the label to fetch values for.")
+	labelValuesCmd.Flags().StringVarP(&labelValuesStart, "start", "s", "", "Start timestamp.")
+	labelValuesCmd.Flags().StringVarP(&labelValuesEnd, "end", "e", "", "End timestamp.")
+
+	err = labelValuesCmd.MarkFlagRequired("name")
+	if err != nil {
+		panic(err)
+	}
+
+	cmd.AddCommand(seriesCmd)
+	cmd.AddCommand(labelsCmd)
+	cmd.AddCommand(labelValuesCmd)
+
+	return cmd
+}
+
+func NewLogsQueryCmd(ctx context.Context) *cobra.Command {
+	var (
+		isRange                     bool
+		time, start, end, direction string
+		limit                       float32
+		step, interval              interface{}
+	)
+	cmd := &cobra.Command{
+		Use:          "query",
+		Short:        "Query metrics for a tenant.",
+		Long:         "Query metrics for a tenant. Can get results for both instant and range queries. Pass a single valid PromQL query to fetch results for.",
+		Example:      `obsctl metrics query "prometheus_http_request_total"`,
```

Comment:
Suggest to replace references of `metrics` with `logs` here as well as `PromQL` with `LogQL`